### PR TITLE
Added initial project structure

### PR DESCRIPTION
**Pull request by JoshuaLight**

### CODE REVIEW

File: candies/cli/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__version__ = 0.0.1
```
Comment (konstantin-ogulchansky) [inline candies/cli/__init__.py]: Shouldn't there be a string? ))

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    url='https://github.com/candy-kingdom/cli',
+    packages=find_packages(exclude=['tests']),
+    include_package_data=True,
+    install_requires=requirements(),
+    classifiers=[
+        'Development Status :: 2 - Pre-Alpha',
+        'Intended Audience :: Developers',
+        'License :: MIT',
+        'Natural Language :: English',
+        'Programming Language :: Python :: 3.7.4',
```
Comment (konstantin-ogulchansky) [inline setup.py]: What do you think about using `3.8.0`?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+from io import open
```
Comment (konstantin-ogulchansky) [inline setup.py]: What do you think about using the built-in `open`?

Comment (JoshuaLight) [reply setup.py]: Changed!

File: candies/cli/__init__.py (hunk 1 above)
Comment (JoshuaLight) [reply candies/cli/__init__.py]: Fixed!

File: setup.py (hunk 2 above)
Comment (JoshuaLight) [reply setup.py]: As far as I know, we can include here any number of languages, and the more we include the more compatible the library is.

Though I'm not sure how many versions we should add, and I'm using `3.7.4` as a laptop default, but we can probably go for `3.8.0` here and then decide (or use some tool), what other versions are supported by the package.

Comment (JoshuaLight) [reply setup.py]: Done!

**Pull request merged**